### PR TITLE
[TR] Translation of 08.md

### PR DESCRIPTION
**Pull request by emirceyani**

### CODE REVIEW

File: docs/tr/week08/08.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+## Ders Kısım B
+
+Bu bölümde, düzenlileştirilmiş saklı değişken EBM'le ve bu modellerin koşullu ve koşulsuz versiyonlarını detaylıca inceledik. 
+Daha sonra, ISTA, FISTA ve LISTA algoritmalarını tartıştık ve seyrek kodlama örnekleri ile evrişimsel seyrek otokodlayıcılardan öğrenilen filtreleri birlikte inceledik.
+Son olarak, değişimsel otokodlayıcılar ve bunun altında yatan temel kavramları ele aldık. 
+
+## Uygulama
+
+Bu bölümde, üretici modellerin özel bir türü olan değişimsel otokodlayıcılardan bahsettik ve bu modelin özellikleri ile avantajlarını klasik otokodlayıcılar ile kıyasladık.
+VAE'nin amaç fonksiyonunu, saklı uzayda nasıl bir yapı oluşturmaya zorladığını anlayarak, detaylica inceledik.
```
Comment (yigitsever) [inline docs/tr/week08/08.md:24]: detaylica -> detaylıca

Comment (yigitsever) [inline docs/tr/week08/08.md:24]:
```suggestion
VAE'nin amaç fonksiyonunu, saklı uzayda nasıl bir yapı garanti ettiğini anlayarak, detaylica inceledik. 
```
The original uses "enforce" which is more akin to "garanti etmek/temennisini vermek"

File: docs/tr/week08/08.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+lang: tr
+translation-date: 14 Jun 2020
+translator: emirceyani
+---
+
+## Ders Kısım A
+
+Bu bölümde, karşıtsal yöntemlerin enerji bazlı modellerdeki(EBM) kullanımına çeşitli açılardan odaklandık.
+Önce, karşıtsal yöntemlerin özdenetimli öğrenmede kullanımının getirdiği avantaj üzerine konuşacağız.
+Sonra, arıtan otokodlayıcıların mimarisini ve bu yapıların bir görüntüyü yeniden oluşturkenki zayıflıklarını ele aldık. Ayrıca, karşıtsal ıraksama(contrastive divergence) ve kalıcı karşıtsal ıraksama(persistent contrastive divergence) gibi diğer karşıtsal yöntemlerden de bahsettik.
```
Comment (yigitsever) [inline docs/tr/week08/08.md:13]:
```suggestion
Sonra, arıtan otokodlayıcıların mimarisini ve bu yapıların bir görüntüyü yeniden oluşturkenki zayıflıklarını ele aldık. Ayrıca, karşıtsal ıraksama (contrastive divergence) ve kalıcı karşıtsal ıraksama (persistent contrastive divergence) gibi diğer karşıtsal yöntemlerden de bahsettik.
```

File: docs/tr/week08/08.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+lang-ref: ch.08
+title: 8. Hafta
+lang: tr
+translation-date: 14 Jun 2020
+translator: emirceyani
+---
+
+## Ders Kısım A
+
+Bu bölümde, karşıtsal yöntemlerin enerji bazlı modellerdeki(EBM) kullanımına çeşitli açılardan odaklandık.
```
Comment (yigitsever) [inline docs/tr/week08/08.md:11]:
```suggestion
Bu bölümde, karşıtsal yöntemlerin enerji bazlı modellerdeki (EBM) kullanımına çeşitli açılardan odaklandık.
```

File: docs/tr/week08/08.md (hunk 1 above)
Comment (emirceyani) [reply docs/tr/week08/08.md:35]: But in reality, it's not a guarantee. IN fact, VAE loss function's KL divergence components acts as a regularizer. If we translate it as 'garanti etmek' then it sounds like it always enforces something. If we think divergence term as a regularizer, which we should actually do, it doesn't always  guarantee a thing at all. You can think enforcing phenomenon like a physical force(Check Yann's VAE explanation with spring balls). In fact you can fail sometimes. "Garanti etmek" is too strong for this sentence I think. We can discuss it on here or Slack.

Comment (yigitsever) [reply docs/tr/week08/08.md:35]: You're absolutely right, thank you for the explanation

Comment (emirceyani) [reply docs/tr/week08/08.md:35]: I also thank you too. I though I was a bit harsh haha

**Comment by Atcold** [comment]
Thanks! Great!

**Pull request merged**